editor/import/resource_importer_imagefont.cpp

```diff
@@ -156,17 +156,16 @@ Error ResourceImporterImageFont::import(const String &p_source_file, const Strin
 								c++; // Skip "+".
 								continue;
 							}
-						} else if (range[c] == '0') {
-							if ((c <= range.length() - 2) && range[c + 1] == 'x') {
-								token = String();
-								if (step == STEP_START_BEGIN) {
-									step = STEP_START_READ_HEX;
-								} else {
-									step = STEP_END_READ_HEX;
-								}
-								c++; // Skip "x".
-								continue;
+						} else if (range[c] == '0' && (c + 1 < range.length()) && range[c + 1] == 'x') {
```

Comment:
```suggestion
						} else if (range[c] == '0' && (c <= range.length() - 2) && range[c + 1] == 'x') {
```
For consistency with the rest.